subjects/java/checkpoints/distinct-substring-lenght/README.md

```diff
@@ -0,0 +1,45 @@
+## Distinct Substring length
+
+### Instructions
+
+Create a class `DistinctSubstringLength` that provides a method to find the length of the longest substring without repeating characters in a given string.
```

Comment:
I would specify if this is case-sensitive or not. In other exercises, the checks where case insensitive while here it seems to be case sensitive (from the tests)

Reply:
the default is to be case-sensitive, but there is no harm to specify that :)